docs/reference/snapshot-restore/repository-gcs.asciidoc

```diff
@@ -191,6 +191,19 @@ are marked as `Secure`.
     can be specified explicitly. For example, it can be used to switch between projects when the
     same credentials are usable for both the production and the development projects.
 
+`proxy.host`::
+    The host name of a proxy to connect to the Google Cloud Storage through.
+    For example: `gcs.client.default.proxy.host: proxy.host`.
```

Comment:
We don't provide examples for other client settings so I'd remove this (also because the host looks wrong :))

Reply:
Good point, let me remove them